src/components/Settings/SettingsLogs/index.tsx

```diff
@@ -1,14 +1,299 @@
+                  clipRule="evenodd"
+                />
+              </svg>
+            </span>
+            <select
+              id="filter"
+              name="filter"
+              onChange={(e) => {
+                setPageIndex(0);
+                setCurrentFilter(e.target.value as Filter);
+              }}
+              value={currentFilter}
+              className="rounded-r-only"
+            >
+              <option value="debug">
+                {intl.formatMessage(messages.filterDebug)}
+              </option>
+              <option value="info">
+                {intl.formatMessage(messages.filterInfo)}
+              </option>
+              <option value="warn">
+                {intl.formatMessage(messages.filterWarn)}
+              </option>
+              <option value="error">
+                {intl.formatMessage(messages.filterError)}
+              </option>
+            </select>
+          </div>
+        </div>
+        <Table>
+          <thead>
+            <tr>
+              <Table.TH>{intl.formatMessage(messages.time)}</Table.TH>
+              <Table.TH>{intl.formatMessage(messages.level)}</Table.TH>
+              <Table.TH>{intl.formatMessage(messages.label)}</Table.TH>
+              <Table.TH>{intl.formatMessage(messages.message)}</Table.TH>
+            </tr>
+          </thead>
+          <Table.TBody>
+            {data.results.map((row: LogMessage, index: number) => {
+              return (
+                <tr className="" key={`log-list-${index}`}>
+                  <Table.TD className="text-gray-300">
+                    {intl.formatDate(row.timestamp, {
+                      year: 'numeric',
+                      month: 'short',
+                      day: '2-digit',
+                      hour: 'numeric',
+                      minute: 'numeric',
+                      second: 'numeric',
+                      hour12: false,
+                    })}
+                  </Table.TD>
+                  <Table.TD className="text-gray-300">
+                    {row.level === 'debug' && (
+                      <Badge badgeType="default">
+                        {row.level.toUpperCase()}
+                      </Badge>
+                    )}
+                    {row.level === 'info' && (
+                      <Badge badgeType="success">
+                        {row.level.toUpperCase()}
+                      </Badge>
+                    )}
+                    {row.level === 'warn' && (
+                      <Badge badgeType="warning">
+                        {row.level.toUpperCase()}
+                      </Badge>
+                    )}
+                    {row.level === 'error' && (
+                      <Badge badgeType="danger">
+                        {row.level.toUpperCase()}
+                      </Badge>
+                    )}
```

Comment:
```suggestion
                    <Badge
                      badgeType={
                        row.level === 'error'
                          ? 'danger'
                          : row.level === 'warn'
                          ? 'warning'
                          : row.level === 'info'
                          ? 'success'
                          : 'default'
                      }
                    >
                      {row.level.toUpperCase()}
                    </Badge>
```

Since only the badge type varies 😺